content/2021-12-14-brigade-2.1.0.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Brigade 2.1.0"
+slug: "2021-12-14-brigade-2.1.0"
```

Comment:
Just needs a bump to 12-15 here and below.

Reply:
Yup! Thanks for the reminder. I would have forgotten!

Reply:
Fixed in 7eea81176a379847df99eab19ae8b6fddd850439